aos/aos_milestones/IMpg/check_impg_run.ipynb

```diff
@@ -0,0 +1,798 @@
+{
```

Comment:
<p>Due to the fact that we use microns for x,y,z and degrees for u,v, if we plot them on the same scale, u and v will be basically indistinguishable from zero..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>done</p>